plugins/cairo-lang-macro/Cargo.toml

```diff
@@ -13,8 +13,8 @@ readme = "README.md"
 repository.workspace = true
 
 [dependencies]
-cairo-lang-macro-attributes = "0.1.1"
-cairo-lang-macro-stable = "1"
+cairo-lang-macro-attributes = { path = "../cairo-lang-macro-attributes" }
+cairo-lang-macro-stable = { path = "../cairo-lang-macro-stable" }
```

Comment:
```suggestion
```
Why change the stable one? 🤔 If we use `NO_OP_ATTR_PREFIX` to differentiate, there is no changed in the interface, right?

Reply:
This is a temporary change so we can run tests, until the new version of `cairo-lang-macro-attributes` is released, as otherwise the same structs are not recognized due to one dependency specified as a path dependency and the other one as a registry dependency:
```
error[E0308]: arguments to this function are incorrect
   --> scarb/src/compiler/plugin/proc_macro/ffi.rs:182:13
    |
182 |             (self.plugin.vtable.expand)(item_name, stable_attr, stable_token_stream);
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^            -----------  ------------------- expected `StableTokenStream`, found a different `StableTokenStream`
    |                                                    |
    |                                                    expected `StableTokenStream`, found a different `StableTokenStream`
    |
    = note: `StableTokenStream` and `StableTokenStream` have similar names, but are actually distinct types
note: `StableTokenStream` is defined in crate `cairo_lang_macro_stable`
   --> /Users/delevo/repos/scarb/plugins/cairo-lang-macro-stable/src/lib.rs:25:1
    |
25  | pub struct StableTokenStream {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: `StableTokenStream` is defined in crate `cairo_lang_macro_stable`
   --> /Users/delevo/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cairo-lang-macro-stable-1.0.0/src/lib.rs:25:1
```  

Afterwards, all changes that include use of path dependencies will be reverted.